vmupdate/qube_connection.py

```diff
@@ -137,19 +139,17 @@ def _copy_file_from_dom0(self, src, dest):
             with open(src, 'rb') as file:
                 proc = Popen(command, stdin=file)
```

Comment:
Not strictly related to this PR, but stdout/stderr should be redirected somewhere (either collected via subprocess.PIPE, or put to DEVNULL), otherwise it may disturb progress bar (at least color it red).
And also, why this one via `qvm-run` instead of python API, as the other calls?